mmv1/third_party/terraform/services/bigquery/iam_bigquery_member_dataset.go

```diff
@@ -0,0 +1,326 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+/*
```

Comment:
This is added to fix https://github.com/GoogleCloudPlatform/magic-modules/pull/10560/files#r1586840421 .